.setup/bin/setup_sample_courses.py

```diff
@@ -192,6 +192,13 @@ def main():
     os.system("crontab -u hwcron /tmp/hwcron_cron_backup.txt")
     os.system("rm /tmp/hwcron_cron_backup.txt")
 
+def generate_random_user_id(length):
```

Comment:
I'd make the parameter have a default value of `15` and then get rid of the 15s below (so it's less of a "magic" number).